### DISCUSSION TITLE
Any way to disable letter highlighting in restults?

### DISCUSSION THREAD

**Discussion by frdrk** [Ulauncher/Ulauncher#903] [Extensions and Themes]
## Question

Is there a way to disable the highlighting of matched letters in the search result when developing an extension? In my case that highlight doesn't fill a functional purpose and only confuses the UX of the extension.

**Comment by friday** [comment]
This isn't covered by the docs, but you pass `highlightable=False` to ExtensionResultItem

https://github.com/brpaz/ulauncher-encoder does this for example.

And if you use the small result view it doesn't highlight either. For example https://github.com/Ulauncher/ulauncher-kill or https://github.com/friday/ulauncher-clipboard

Keeping this issue open until it's documented.
